modules/logging-create-clf.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/log-forwarding.adoc
+
+:_content-type: REFERENCE
+[id="logging-create-clf_{context}"]
+= Creating a log forwarder
+
+To create a log forwarder, you must create a `ClusterLogForwarder` CR that specifies the log input types that the service account can collect. You can also specify which outputs the logs can be forwarded to. If you are using multi log forwarder mode, you must also reference the service account in the `ClusterLogForwarder` CR.
+
+If you are using multi log forwarder mode on your cluster, you can create `ClusterLogForwarder` custom resources (CRs) in any namespace, using any name.
```

Comment:
I think there is one more thing we have to write down: if users want to create CL/CLF in any namespace, they must deploy CLO to watch all namespaces in the cluster.

Reply:
I think that this was previously discussed with @jcantrill , and the outcome was that we assume the installation is watching all namespaces?

Reply:
See https://github.com/openshift/openshift-docs/pull/64935#discussion_r1342652193